docs/installing.rst

```diff
@@ -28,6 +28,10 @@ From here, you can now include ``extern/pybind11/include``, or you can use
 the various integration tools (see :ref:`compiling`) pybind11 provides directly
 from the local folder.
```

Comment:
I think this is the correct cross reference, there you find detail on how to work with this. It was not supposed to mean that "python3 -m pybind11 --includes" is supposed to work when included as a submodule. Maybe a note somewhere mentioning that `python3 -m pybind11 --includes` is only available when you install `pybind11`?

Reply:
Thanks @henryiii, that works for me.  As a first-time visitor to the doc, I choose the first method of "Installing the library", i.e., "Include as a submodule", and then followed the "First steps" section of the doc.  That's when I thought to set PYTHONPATH to make the demo work.

But I agree that it is not a good idea to bring PYTHONPATH into the discussion if it is not needed.

Reply:
I think we should instead tweak the installing docs to make it clear that installing should be required to make `python -m pybind11` work? I would have thought that was implicit.

PS: Due to the support for the situation I listed above, setting `PYTHONPATH` does work, and isn't likely to stop working, I don't think we should tell people about it, but if it works for you _and_ you are the end user (not expecting anyone else to do that), then go ahead and use it!